project/frontend/src/components/PreferenceForm.js

```diff
@@ -120,6 +126,21 @@ function PreferenceForm(props) {
     );
   };
 
+  const setTooltipVal = (val, tooltipInd) => {
```

Comment:
If you go with the array param route I talked about above, this is not applicable, but i feel like `tooltipInd` should be either `index`, `tooltipIdx`, or `tooltipIndex`. Idk if this is just me but `Ind` feels like a weird abbreviation of index.